fluent-syntax/src/ast.js

```diff
@@ -179,10 +185,10 @@ export class Function extends Identifier {
   }
 }
 
-export class JunkEntry extends Entry {
+export class Junk extends Entry {
   constructor(content) {
     super();
-    this.type = 'JunkEntry';
+    this.type = 'Junk';
     this.content = content;
```

Comment:
With each `Entry` now storing its `span`, is it OK to additionally store `content` in `Junk`?